src/sheets/quadrone/actor/tabs/SidebarTabTraits.svelte

```diff
@@ -0,0 +1,147 @@
+<script lang="ts">
+  import { FoundryAdapter } from 'src/foundry/foundry-adapter';
+  import ActorTraitConfigurableListEntry from '../parts/ActorTraitConfigurableListEntry.svelte';
+  import { getCharacterSheetQuadroneContext } from 'src/sheets/sheet-context.svelte';
+  import { SpecialTraitsApplication } from 'src/applications-quadrone/special-traits/SpecialTraitsApplication.svelte';
+  import NpcTraitCreatureType from '../npc-parts/traits/NpcTraitCreatureType.svelte';
+  import NpcTraitSize from '../npc-parts/traits/NpcTraitSize.svelte';
+
+  let context = $derived(getCharacterSheetQuadroneContext());
+
+  const localize = FoundryAdapter.localize;
+</script>
+
+<div class="list traits">
+  <!-- Speed -->
+  <ActorTraitConfigurableListEntry
+    configButtonLocation="label"
+    label={localize('DND5E.Speed')}
+    entries={context.speeds.traitEntries}
+    onconfig={() =>
+      FoundryAdapter.renderMovementSensesConfig(context.actor, 'movement')}
+    icon="fa-solid fa-rabbit-running"
+  />
+
+  <!-- Senses -->
+  <ActorTraitConfigurableListEntry
+    configButtonLocation="label"
+    label={localize('DND5E.Senses')}
+    entries={context.senses.traitEntries}
+    onconfig={() =>
+      FoundryAdapter.renderMovementSensesConfig(context.actor, 'senses')}
+    icon="fa-solid fa-eye"
+  />
+
+  <!-- Size -->
+  {#if context.unlocked}
+    <NpcTraitSize />
```

Comment:
`ActorTraitSize`, if possible.
Though the tab files are not named for it, currently, this is going to be geared toward just the Character if I understand.